practice-one/src/practice-one.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+.nav{
+    align-items: flex-start;
+    display: flex;
+    height: 43px;
+    width: 1328px;
+    margin-top: 40px;
+   
+}
+.frame{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    position: absolute;
+    width: 500px;
+    height: 40px;
+    left: 0px;
+    top: 40px;
+}
+.open-enterprise{
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: -0.04em;
+    margin-left: 125px;
+    height: 38px;
+    max-width: 400px;
+    font-weight: 400;
+    font-size: 32px;
+    margin-top: 30px;
+}
+.logo{
+    position: absolute;
+    left: 56px;
+    height: 40px;
+    width: 68px;
+    margin-top: 30px;
+    float: left;
+  
+}
+.nav-link{
+    float: right;
+    list-style: none;
+    margin-top: 30px;
+    margin-right: -1400px;
+}
+.nav-link li{
+    display: inline-block;
+    margin-right: 30px;
+}
+.nav-link li a{
```

Comment:
Avoid style in element tags, should add class for styling

Reply:
fixed